quic/s2n-quic-core/src/connection/limits.rs

```diff
@@ -170,10 +215,13 @@ impl Limits {
     }
 
     #[doc(hidden)]
-    pub const fn initial_flow_control_limits(&self) -> InitialFlowControlLimits {
+    pub fn initial_flow_control_limits(&self) -> InitialFlowControlLimits {
         InitialFlowControlLimits {
             stream_limits: self.initial_stream_limits(),
             max_data: self.data_window.as_varint(),
+            // TODO max_open_remote_bidirectional_streams is unused at the moment.
+            // Use it once the the stream controller is initiator aware.
+            // https://github.com/aws/s2n-quic/issues/1388
```

Comment:
Why not just use the new value?

Reply:
The stream controller is not initiator aware at the moment. Using these values will change that behavior which I intend to do in a separate PR. The goal of this PR is to simply expose the API, which will unlock testing.

The Option is needed if we plan on maintaining backwards behavior of customers using `max_open_bidirectional_streams`.. we need a way to detect if customers called the new api, otherwise default `max_open_bidirectional_streams`.